pkg/services/alerting/conditions/query.go

```diff
@@ -124,7 +171,10 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange p
 		return nil, fmt.Errorf("access denied: %w", err)
 	}
 
-	req := c.getRequestForAlertRule(getDsInfo.Result, timeRange, context.IsDebug)
+	req, err := c.getRequestForAlertRule(getDsInfo.Result, timeRange, context.IsDebug)
```

Comment:
this function can now return an error, this could not happen in the past. i wonder if that is ok, or we should make this never-fail ( we could just set a default-value for `.Interval` in error-case maybe?)